docs/coordinates/skycoord.rst

```diff
@@ -1199,6 +1199,88 @@ example::
   >>> FK4() == FK4(obstime='2020-01-01')
   False
 
+.. _skycoord-to-table:
```

Comment:
Perhaps this new section should adjoin / be merged with the ``SkyCoord.guess_from_table`` docs.

Reply:
That would be the best place, but `SkyCoord.guess_from_table()` does not seem to have any documentation outside its docstring and a couple of change log entries.

Reply:
Good notice. That's certainly not great. Do you want to raise an issue to add documentation on ``guess_from_table``? It would go HERE. In that case, I suggest changing the RST link 

```suggestion
.. _skycoord-to_from-table:
```
Or something similar. Or hold off until ``guess_from_table`` is documented.

Reply:
If that documentation hasn't been written so far then maybe we shouldn't expect it to be written anytime soon.

Reply:
Well, there wasn't a strong need because it's documented in the docstring. But now that there's a fleshed out SkyCoord <-> Table, and a section on SkyCoord -> Table. I can raise the issue, if you don't want to. No worries.
I did some work on ``guess_from_table`` a couple months back, so I'm happy to assist and/or contribute to writing the section.

Reply:
I you are willing to write that documentation then it does make sense to rename the RST link.